src/NuGet.Core/NuGet.CommandLine.XPlat/ReportRenderers/ConsoleRenderer/ListPackageConsoleRenderer.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGet.CommandLine.XPlat
+{
+    /// <summary>
+    /// Console output renderer for dotnet list package command
+    /// </summary>
+    internal class ListPackageConsoleRenderer : IReportRenderer
+    {
+        protected List<ReportProblem> _problems = new();
+
+        private ListPackageConsoleRenderer()
+        { }
+
+        public static ListPackageConsoleRenderer GetInstance()
+        {
+            return new ListPackageConsoleRenderer();
+        }
```

Comment:
What's the benefit of this factory method, vs just using `new ListPackageConsoleRenderer()` at the same call site?

Reply:
It's just my personal preference, I replaced them with `new ListPackageConsoleRenderer()` at the call sites.